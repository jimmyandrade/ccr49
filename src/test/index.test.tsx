import IndexPage from "../pages";
import {create} from "react-test-renderer";

jest.mock("next/image", () => "Image");

describe("Index page", () => {
    it("should match snapshot", () => {
        const testRenderer = create(<IndexPage />);
        expect(testRenderer).toMatchSnapshot();
    })
})