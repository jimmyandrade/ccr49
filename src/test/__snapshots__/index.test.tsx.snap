// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index page should match snapshot 1`] = `
Array [
  <div
    className="MuiBox-root MuiBox-root-1"
  >
    <header
      className="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorDefault MuiPaper-elevation4"
    >
      <div
        className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
      >
        <div
          className="MuiBox-root MuiBox-root-2"
        >
          <a
            className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
            href="/"
            onBlur={[Function]}
            onClick={[Function]}
            onFocus={[Function]}
            onMouseEnter={[Function]}
          >
            <h1
              className="MuiTypography-root MuiTypography-srOnly"
            >
              Jornada UOU
            </h1>
            <svg
              className="Logo-root-3"
              data-name="Camada 1"
              id="Camada_1"
              viewBox="0 0 160 93"
              xmlns="http://www.w3.org/2000/svg"
            >
              <title>
                Logotipo de Jornada UOU
              </title>
              <path
                className="Logo-svgClass1-4"
                d="M86.88,16.74l2.46-1.47c.54-.33,1-.3,1.3.32a1.45,1.45,0,0,0,1,.9c.37.1.67.53,1,.85s.53.43.9.14a5.08,5.08,0,0,1,.75-.44c.81-.46,1.28-.18,1.28.75a4.61,4.61,0,0,0,.65,2.79c.68-.39,1,.06,1.34.57.24.35.65.59.89.94a14.22,14.22,0,0,1,.76,1.45c1-.18,1-.09,1.3.93s.84,1.86,1.19,2.82a5.53,5.53,0,0,1,.13,1.42c.88-.12,1.21.2,1,1.22a1.73,1.73,0,0,0,.22,1.43c.15.22.06.6.11.91s.19.72.23,1.09a1.87,1.87,0,0,1-.06.82c-.2.5-.46,1-.7,1.43-.45.85-.26,1.34.58,1.9a1,1,0,0,1,.32.84c-.2,1.26-.48,2.5-.72,3.76a.48.48,0,0,0,.07.38c.46.43.27.81,0,1.2l-1.11,1.42c.7.47.77.76.23,1.36-.91,1-1.88,2-2.81,3a2.35,2.35,0,0,0-.42.78A2.5,2.5,0,0,1,97,51.33c-.54-.11-.67-.69-.23-1.19.68-.78,1.4-1.52,2.14-2.31A1,1,0,0,1,99,46.46a5.37,5.37,0,0,1,.63-.78,6.67,6.67,0,0,0,2-4.2,21.73,21.73,0,0,0,.45-2.64,2.24,2.24,0,0,0-.89-1.44c-.45-.37-.55-.67-.3-1.15.15-.28.27-.58.43-.84a3.65,3.65,0,0,0,.31-3.48,5.07,5.07,0,0,1,0-1.52,4.44,4.44,0,0,1-.74.34c-.21,0-.52.09-.65,0a1,1,0,0,1-.25-.67,1.9,1.9,0,0,1,.19-.77,2.62,2.62,0,0,0-.5-2.72,4.78,4.78,0,0,1-.43-1.41c-.64.2-.88-.2-1.16-.75A25.78,25.78,0,0,0,96.54,22c-1.19.24-1.19.24-1.57-.9,0-.11,0-.29-.13-.33-.89-.51-.53-1.39-.63-2.13-.06,0-.11-.06-.14,0-1.37.61-1.36.6-2.45-.45-.16-.15-.44-.16-.64-.27a2.1,2.1,0,0,1-.68-.44,6.53,6.53,0,0,1-.58-.94l-2.28,1.32c-.86.5-1.16.47-1.48-.43a1.93,1.93,0,0,0-2.12-1.36c-1.74-.06-3.49.3-5.23-.12a1.19,1.19,0,0,0-.73.2c-4.23,2.2-7.91,5.22-11.72,8.06a2.17,2.17,0,0,0-.4.56c-.17.28-.33.58-.51.86C64.4,27,63.54,28.47,62.8,30a2,2,0,0,1,.43-.15c.24,0,.58,0,.7.1a1.06,1.06,0,0,1,.12.77,3.67,3.67,0,0,1-.38.77l-1.35,3c.52.1,1,.05,1.15.25a1.78,1.78,0,0,1,.11,1.09c0,.48-.34,1,.21,1.39.12.09,0,.6-.11.85-.31.6-.69,1.17-.94,1.59A9.45,9.45,0,0,1,63,41.13a6.41,6.41,0,0,0,.81,3.11c.43.76.7,1.6,1.08,2.38.17.33.47.58.63.91a1.75,1.75,0,0,1,.2.86,9.72,9.72,0,0,1-.26,1.34c.75.26,1.49.48,2.2.78a1.27,1.27,0,0,1,.62.63,3.3,3.3,0,0,1,.08,1c1-.06,1-.06,1.34,2.14,1.58-.39,3.26.22,4.81-.62.52-.29,1.08,0,1,.58-.23,1.36.72,1.91,1.51,2.63.61-.36,1.21-.69,1.79-1.07s.87-.33,1.2.13a2,2,0,0,0,2,.83c.69-.08,1.39.05,2.08,0s1.09-.25,1.64-.3a3,3,0,0,1,.88.16c-.07.29,0,.75-.22.84a4.32,4.32,0,0,1-1.64.56c-1,.08-1.91,0-2.87,0a.22.22,0,0,0-.14,0c-1.1.67-1.64-.18-2.26-.83a3,3,0,0,0-.24-.18c-.61.35-1.22.68-1.8,1a.86.86,0,0,1-1.16-.12c-.5-.43-.85-1.12-1.68-1.11-.11,0-.29-.39-.34-.63a4.88,4.88,0,0,1,0-1c-1.65.1-3.22.16-4.78.31-.72.07-.9,0-1-.69,0-.33-.09-.67-.14-1-1.11.25-1.26.13-1.25-1,0-.57.12-1.17-.66-1.38-.51-.12-.9-.6-1.54-.25-.44.25-.79,0-.82-.56a3,3,0,0,1,.15-.92A2.44,2.44,0,0,0,63.38,47a.4.4,0,0,1-.13-.18c-.36-1-.72-2-1.07-3-.23-.65-.42-1.3-.65-2a.24.24,0,0,1-.2,0c-.25,0-.6,0-.7-.19a1,1,0,0,1,0-.79,3.85,3.85,0,0,1,.59-1.07,5,5,0,0,0,.93-3.23s-.08-.06-.12,0c-.33,0-.85.17-1,0a1.83,1.83,0,0,1-.12-1.06,9.06,9.06,0,0,1,.17-1.73,10.31,10.31,0,0,1,.62-1.46s-.06-.06-.09,0c-.38.13-.76.55-1.17.1s0-.9.18-1.31a15.48,15.48,0,0,0,.94-1.62,10.93,10.93,0,0,1,1.66-3.13.4.4,0,0,0,0-.37c-.4-.54-.13-.93.27-1.24,1.64-1.25,3.3-2.48,4.94-3.72.19-.14.36-.31.55-.48-.25-.43-.39-.84.19-1.18,1.53-.89,3-1.87,4.59-2.68,1.36-.7,2.82-1.16,4.17-1.92.55-.32,1.48.08,2.24.1.56,0,1.11-.13,1.66-.13s1.38.07,2.07.11l.14,0C85.39,14.81,86.67,14.92,86.88,16.74Z"
              />
              <path
                className="Logo-svgClass2-5"
                d="M71.64,57.71l4.61.06L73,64.5h0l8.06-6.66,3.48,0L72.79,67.5l-4.59-.07,3.21-6.72h0l-8,6.65-3.48,0Z"
              />
              <path
                className="Logo-svgClass2-5"
                d="M61.86,36.1C59.22,48.87,56.11,58.25,41.5,58.25c-9,0-16.11-3.23-16.11-12.66A35.17,35.17,0,0,1,26.25,38L30.8,17.4h0A8.81,8.81,0,0,1,39.41,28L36.84,40.12a27.93,27.93,0,0,0-.75,4.38c0,3.28,1.9,5.81,5.7,5.81,6.9,0,8-8.29,9.37-14.73l4-18.18h0A8.65,8.65,0,0,1,63.6,27.82Z"
              />
              <path
                className="Logo-svgClass2-5"
                d="M139.77,37.69c-2.44,12.81-5.39,22.24-20,22.48-9,.14-16.16-3-16.32-12.4a35.15,35.15,0,0,1,.74-7.6L108.4,19.5h0A8.8,8.8,0,0,1,117.18,30l-2.37,12.13a28.51,28.51,0,0,0-.68,4.39c.06,3.27,2,5.77,5.79,5.71,6.91-.11,7.86-8.41,9.14-14.88l3.67-18.24h0a8.64,8.64,0,0,1,8.63,10.28Z"
              />
              <path
                className="Logo-svgClass2-5"
                d="M26.1,63.14c-2.1,1.7-3.67,3.07-6.53,3.56a15.82,15.82,0,0,1-2.58.14,9.37,9.37,0,0,1-2.4-.36l2.66-2.07a3.7,3.7,0,0,0,1.44.27c1.9,0,3.4-1.35,4.6-2.33l6.46-5.26,3.74,0Z"
              />
              <path
                className="Logo-svgClass2-5"
                d="M33.34,67.08c-4.49-.06-4.35-2-2.54-4a18.24,18.24,0,0,1,13.61-5.94c4.49.07,4.35,2,2.53,4A18.2,18.2,0,0,1,33.34,67.08Zm9.08-8c-2.85,0-6.14,2.25-7.64,3.88-1.11,1.21-1.38,2.23.54,2.26,2.85,0,6.15-2.25,7.63-3.86C44.06,60.1,44.34,59.07,42.42,59Z"
              />
              <path
                className="Logo-svgClass2-5"
                d="M56.07,57.48l4.41.06c2.62.05,5.73.16,3.55,2.55-1.56,1.7-4,2.26-6.28,2.41l0,0c1,.08.45,1.11.07,1.74l-1.72,3-3.68,0,1.53-2.47c.8-1.27.77-1.36-.89-1.39h-.4L48,67.13l-3.7,0Zm-1.24,4,1.25,0c1.6,0,3.1-.24,3.94-1.15.63-.69.3-1-.79-1l-1.76,0Z"
              />
              <path
                className="Logo-svgClass2-5"
                d="M93.6,58l4.63.07-6.57,9.68-3.72-.06,1.51-2.06-5.36-.08-3.53,2-4-.06Zm0,2h0L87.2,63.78l3.56,0Z"
              />
              <path
                className="Logo-svgClass2-5"
                d="M107.2,58.23l4.79.07c4.39.07,5.85,1.41,3.5,4a18.39,18.39,0,0,1-14.09,5.64l-6-.08ZM101.5,66l1.62,0a11.66,11.66,0,0,0,8.41-3.62c1.61-1.77.21-2.17-2-2.2h-.91Z"
              />
              <path
                className="Logo-svgClass2-5"
                d="M129.14,58.56l4.63.07-6.57,9.68-3.72-.06L125,66.19l-5.36-.08-3.53,2-4-.06Zm0,2h0l-6.38,3.73,3.56.06Z"
              />
              <polyline
                className="Logo-svgClass2-5"
                points="65.27 62.91 71.56 57.62 76.55 57.62 74.21 61.94 70.15 63.35 71.87 60.42 68.19 63.35"
              />
            </svg>
          </a>
        </div>
        <button
          className="MuiButtonBase-root MuiIconButton-root"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
        >
          <span
            className="MuiIconButton-label"
          >
            <div
              className="MuiAvatar-root MuiAvatar-circular MuiAvatar-colorDefault"
            >
              <Image
                alt="Imagem de perfil de Jornada UOU"
                height={40}
                src="test-file-stub"
                width={40}
              />
            </div>
          </span>
        </button>
      </div>
    </header>
  </div>,
  <main
    className="MuiContainer-root MuiContainer-maxWidthLg"
  >
    <h1
      className="MuiTypography-root MuiTypography-srOnly"
    >
      Página inicial
    </h1>
    <div
      className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 MuiGrid-justify-content-xs-center"
    >
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6"
      >
        <div
          className="MuiPaper-root MuiCard-root MuiBox-root MuiBox-root-7 MuiPaper-elevation1 MuiPaper-rounded"
        >
          <div
            className="MuiCardMedia-root"
          >
            <Image
              src="test-file-stub"
            />
          </div>
          <div
            className="MuiCardContent-root"
          >
            <h2
              className="MuiTypography-root MuiTypography-h4 MuiTypography-gutterBottom"
            >
              Trilhas
            </h2>
            <p
              className="MuiTypography-root MuiTypography-body2 MuiTypography-paragraph"
            >
              Conteúdos agrupados e ordenados para facilitar sua aprendizagem
            </p>
            <a
              aria-disabled={false}
              className="MuiButtonBase-root MuiButton-root MuiButton-outlined"
              href="/trilhas"
              onBlur={[Function]}
              onClick={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseEnter={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
            >
              <span
                className="MuiButton-label"
              >
                Ver trilhas
              </span>
            </a>
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6"
      >
        <div
          className="MuiPaper-root MuiCard-root MuiBox-root MuiBox-root-8 MuiPaper-elevation1 MuiPaper-rounded"
        >
          <div
            className="MuiCardMedia-root"
          >
            <Image
              src="test-file-stub"
            />
          </div>
          <div
            className="MuiCardContent-root"
          >
            <h2
              className="MuiTypography-root MuiTypography-h4 MuiTypography-gutterBottom"
            >
              Anuncie aqui
            </h2>
            <p
              className="MuiTypography-root MuiTypography-body2 MuiTypography-paragraph"
            >
              Ajude a manter nossa plataforma e apoiar jovens de todo o Brasil.
            </p>
            <a
              aria-disabled={false}
              className="MuiButtonBase-root MuiButton-root MuiButton-outlined"
              href="#"
              onBlur={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
            >
              <span
                className="MuiButton-label"
              >
                Quero anunciar
              </span>
            </a>
          </div>
        </div>
      </div>
    </div>
    <div
      className="MuiBox-root MuiBox-root-9"
    >
      <div
        className="MuiBox-root MuiBox-root-10"
      >
        <p
          className="MuiTypography-root MuiTypography-overline"
        >
          Benefícios de 
          <strong>
            lazer
          </strong>
          , 
          <strong>
            cultura
          </strong>
           e oportunidades
        </p>
        <span
          className="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom"
        >
          Bônus UOU
        </span>
      </div>
      <div
        className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 MuiGrid-justify-content-xs-center"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-3"
        >
          <div
            className="MuiPaper-root MuiCard-root MuiBox-root MuiBox-root-11 MuiPaper-elevation1 MuiPaper-rounded"
          >
            <div
              className="MuiCardMedia-root"
            >
              <Image
                src="test-file-stub"
              />
            </div>
            <div
              className="MuiCardContent-root"
            >
              <div
                className="MuiBox-root MuiBox-root-12"
              >
                <div
                  className="MuiAvatar-root MuiAvatar-circular ProfileAvatar-large-13 MuiAvatar-colorDefault"
                >
                  <svg
                    aria-hidden={true}
                    className="MuiSvgIcon-root"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M18 4l2 4h-3l-2-4h-2l2 4h-3l-2-4H8l2 4H7L5 4H4c-1.1 0-1.99.9-1.99 2L2 18c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V4h-4z"
                    />
                  </svg>
                </div>
              </div>
              <h3
                className="MuiTypography-root MuiTypography-h6 MuiTypography-gutterBottom"
              >
                Cinema gratuito ou com desconto
              </h3>
              <p
                className="MuiTypography-root MuiTypography-body2 MuiTypography-gutterBottom MuiTypography-paragraph"
              >
                <strong>
                  Parceria
                </strong>
                 com a 
                <strong>
                  Cineart
                </strong>
                , presente em vários 
                <i
                  lang="en"
                >
                  shoppings
                </i>
                 do Brasil.
              </p>
            </div>
          </div>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-3"
        >
          <div
            className="MuiPaper-root MuiCard-root MuiBox-root MuiBox-root-14 MuiPaper-elevation1 MuiPaper-rounded"
          >
            <div
              className="MuiCardMedia-root"
            >
              <Image
                src="test-file-stub"
              />
            </div>
            <div
              className="MuiCardContent-root"
            >
              <div
                className="MuiBox-root MuiBox-root-15"
              >
                <div
                  className="MuiAvatar-root MuiAvatar-circular ProfileAvatar-large-16 MuiAvatar-colorDefault"
                >
                  <svg
                    aria-hidden={true}
                    className="MuiSvgIcon-root"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M22 10V6c0-1.11-.9-2-2-2H4c-1.1 0-1.99.89-1.99 2v4c1.1 0 1.99.9 1.99 2s-.89 2-2 2v4c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2v-4c-1.1 0-2-.9-2-2s.9-2 2-2zm-9 7.5h-2v-2h2v2zm0-4.5h-2v-2h2v2zm0-4.5h-2v-2h2v2z"
                    />
                  </svg>
                </div>
              </div>
              <h3
                className="MuiTypography-root MuiTypography-h6 MuiTypography-gutterBottom"
              >
                Acesso a shows e peças de teatro
              </h3>
              <p
                className="MuiTypography-root MuiTypography-body2 MuiTypography-gutterBottom MuiTypography-paragraph"
              >
                Pague mais barato ou até entre de graça em 
                <strong>
                  parceria
                </strong>
                 com a Rede 
                <abbr
                  title="Serviço Social do Comércio"
                >
                  SESC
                </abbr>
                .
              </p>
            </div>
          </div>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-3"
        >
          <div
            className="MuiPaper-root MuiCard-root MuiBox-root MuiBox-root-17 MuiPaper-elevation1 MuiPaper-rounded"
          >
            <div
              className="MuiCardContent-root"
            >
              <div
                className="MuiBox-root MuiBox-root-18"
              >
                <div
                  className="MuiAvatar-root MuiAvatar-circular ProfileAvatar-large-19 MuiAvatar-colorDefault"
                >
                  <svg
                    aria-hidden={true}
                    className="MuiSvgIcon-root"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M12 11.55C9.64 9.35 6.48 8 3 8v11c3.48 0 6.64 1.35 9 3.55 2.36-2.19 5.52-3.55 9-3.55V8c-3.48 0-6.64 1.35-9 3.55zM12 8c1.66 0 3-1.34 3-3s-1.34-3-3-3-3 1.34-3 3 1.34 3 3 3z"
                    />
                  </svg>
                </div>
              </div>
              <h3
                className="MuiTypography-root MuiTypography-h6 MuiTypography-gutterBottom"
              >
                Desconto na compra de livros
              </h3>
              <p
                className="MuiTypography-root MuiTypography-body2 MuiTypography-gutterBottom MuiTypography-paragraph"
              >
                Em parceria com a 
                <strong>
                  Livraria Cultura
                </strong>
                , você tem desconto na compra de livros.
              </p>
            </div>
          </div>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-3"
        >
          <div
            className="MuiPaper-root MuiCard-root MuiBox-root MuiBox-root-20 MuiPaper-elevation1 MuiPaper-rounded"
          >
            <div
              className="MuiCardContent-root"
            >
              <div
                className="MuiBox-root MuiBox-root-21"
              >
                <div
                  className="MuiAvatar-root MuiAvatar-circular ProfileAvatar-large-22 MuiAvatar-colorDefault"
                >
                  <svg
                    aria-hidden={true}
                    className="MuiSvgIcon-root"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M16.67 13.13C18.04 14.06 19 15.32 19 17v3h4v-3c0-2.18-3.57-3.47-6.33-3.87z"
                      fillRule="evenodd"
                    />
                    <circle
                      cx="9"
                      cy="8"
                      fillRule="evenodd"
                      r="4"
                    />
                    <path
                      d="M15 12c2.21 0 4-1.79 4-4s-1.79-4-4-4c-.47 0-.91.1-1.33.24C14.5 5.27 15 6.58 15 8s-.5 2.73-1.33 3.76c.42.14.86.24 1.33.24zM9 13c-2.67 0-8 1.34-8 4v3h16v-3c0-2.66-5.33-4-8-4z"
                      fillRule="evenodd"
                    />
                  </svg>
                </div>
              </div>
              <h3
                className="MuiTypography-root MuiTypography-h6 MuiTypography-gutterBottom"
              >
                Indicação para oportunidades
              </h3>
              <p
                className="MuiTypography-root MuiTypography-body2 MuiTypography-gutterBottom MuiTypography-paragraph"
              >
                Podemos indicar você para um curso, dependendo do seu propósito.
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </main>,
  <div
    className="MuiBox-root MuiBox-root-23"
  >
    <footer
      className="MuiBox-root MuiBox-root-25 Footer-root-24"
    >
      <div
        className="MuiContainer-root MuiBox-root MuiBox-root-26 MuiContainer-maxWidthLg"
      >
        <p
          className="MuiTypography-root MuiTypography-caption MuiTypography-paragraph"
        >
          <time
            dateTime="2021"
          >
            2021
          </time>
           
          <strong>
            Jornada UOU
          </strong>
           
           - Valorize seu caminho
          <br />
          Crescimento. Evolução. Autonomia.
        </p>
        <p
          className="MuiTypography-root MuiTypography-caption MuiTypography-paragraph"
        >
          <a
            className="MuiTypography-root MuiLink-root MuiLink-underlineAlways MuiTypography-colorInherit"
            href="#"
            onBlur={[Function]}
            onFocus={[Function]}
          >
            Política de privacidade
          </a>
           
          <a
            className="MuiTypography-root MuiLink-root MuiLink-underlineAlways MuiTypography-colorInherit"
            href="#"
            onBlur={[Function]}
            onFocus={[Function]}
          >
            Proteção e uso de dados
          </a>
        </p>
      </div>
    </footer>
  </div>,
]
`;
